examples/jitsi/jitsi-server.yaml

```diff
@@ -0,0 +1,476 @@
+# Configmaps
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prosody-common
+  labels:
+    app.kubernetes.io/name: jitsi-meet
+    app.kubernetes.io/instance: jitsi
+data:
+  ENABLE_AUTH: "0"
+  ENABLE_GUESTS: "1"
+  ENABLE_P2P: "0"
+  PUBLIC_URL: "https://<public-ingress-ip>.nip.io" # TODO: Change this
+  TURN_ENABLE: "true"
+  TURN_HOST: "<stunner-public-ip>" # TODO: Change this
+  TURN_PORT: "3478"
+  TURN_CREDENTIALS: "my-shared-secret"
+  TURN_TRANSPORT: "udp"
+  JICOFO_AUTH_USER: "focus"
+  XMPP_DOMAIN: <public-ingress-ip>.nip.io
+  XMPP_MUC_DOMAIN: muc.<public-ingress-ip>.nip.io
+  XMPP_AUTH_DOMAIN: auth.<public-ingress-ip>.nip.io
+  XMPP_GUEST_DOMAIN: guest.<public-ingress-ip>.nip.io
+  XMPP_RECORDER_DOMAIN: recorder.<public-ingress-ip>.nip.io
+  XMPP_INTERNAL_MUC_DOMAIN: internal-muc.<public-ingress-ip>.nip.io
+  ENABLE_XMPP_WEBSOCKET: "1"
+  ENABLE_COLIBRI_WEBSOCKET: "1"
+  XMPP_MUC_MODULES: "external_services"
+  TZ: 'UTC'
+  LOG_LEVEL: 'debug'
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jitsi-meet
+  labels:
+    app.kubernetes.io/name: jitsi-meet
+    app.kubernetes.io/instance: jitsi
+    app.kubernetes.io/component: web
+data:
+  DISABLE_HTTPS: "1"
+  JICOFO_AUTH_USER: 'focus'
+  XMPP_BOSH_URL_BASE: 'http://jitsi-prosody.default.svc.cluster.local:5280'
+  NGINX_RESOLVER: '<kube-dns>' # TODO: Change this
+  DEPLOYMENTINFO_REGION: 'Europe/Budapest'
```

Comment:
Do we absolutely need this? Do we have at least one user (apart from us) from Hungary?...:-)

Reply:
Fixed in https://github.com/l7mp/stunner/commit/f14c9110a86f67cad68ded4b8cf6693bb6f45628